lib/Doctrine/Common/Cache/CouchbaseBucketCache.php

```diff
@@ -186,15 +182,14 @@ protected function doGetStats()
     }
 
     /**
-     * @param $id
-     * @return bool|string
+     * @param string $id
+     * @return string
      */
-    private function normalizeKey($id)
+    private function normalizeKey(string $id)
```

Comment:
It should be `private function normalizeKey(string $id) : string` instead, right?